doc/UsersGuide/source/Backgroundinfo/TechnicalOverview.rst

```diff
@@ -91,3 +91,151 @@ HAFS Workflow Schematic
+
+Scientific Changes to Improve Track and Intensity Forecast Skills
+-----------------------------------------------------------------
+
+**System and Infrastructure Upgrades:**
+    - Latest version of ufs-weather-model, HAFSv2 final scientific configuration freeze on 02/08/2024.
+    - Increase moving nesting horizontal resolution from 6-2 km to 5.4-1.8 km (HFSA only).
+    - Reduce model time step from 90 to 72s (HFSB only).
+    - Updated horizontal advection options.
+    - Improved model stability and runtime efficiency.
+
+**Vortex Initialization Improvement:**
+    - Enhance vortex initialization to cycle hydrometeor variables and vertical velocity (HFSA only).
+    - Update compsitie vortex and reduce warm-cycling Vmax threshold from 50 to 40 kt (HFSA only).
+
+**Data Assimilation Improvement:**
+    - Ingest new high-resolution GOES-R mesoscale AMVs.
+    - Scale-Dependent Localization for innercore DA.
+    - Refine GPS RO (Radio Occultation) DA.
+
+**Model Physics Advancement:**
+    - Upgrade Thompson MP with bug fixes.
+    - Thompson Microphysics for NATL basin, GFDL Microphysics for EPAC/CPAC and JTWC basins (HFSA only).
+    - Update TKE EDMF PBL and SASAS CP schemes with vertical wind shear impacts.
+    - Change the radiation calling time step from 720s to 900s (HFSA only).
+    - Reduce radiation time step from 1800 to 720s (HFSB only).
+    - Update CO2 fix files.
+
+**Ocean/Wave Coupling:**
+    - MOM6 ocean model coupling (HFSA only).
+    - CMEPS with inline-CDEPS coupling (HFSA only).
+
+**Post-processing:**
+    - Upgrade GFDL Tracker.
+
+**Software Dependencies:**
+    - See `here <https://github.com/hafs-community/HAFS/blob/production/hafs.v2/versions/run.ver>`__
+
+**Output Changes:**
+    - See `here <https://docs.google.com/presentation/d/1dGWu_k-CdiX_ndaRe89iDznyNwkEJfjVTagvvEVZQaw/edit?usp=sharing>`__
+
+Computer Resource Information
+-----------------------------
+
+**Computation resource updates:**
+    - See `here <https://docs.google.com/presentation/d/1otBHAi3hfB1Vu5lk9bfjOE-GZYalZiu9wjnFlevQRQw/edit?usp=sharing>`__
+
+**List of the module versions used in HAFS:**
+    - See `here <https://github.com/hafs-community/HAFS/blob/production/hafs.v2/versions/run.ver>`__
+
+**Data retention for files in:** 
+    - ``/lfs/h1/ops/prod/com/hafs/v2.0/hfsa.yyyymmdd/hh`` and ``/lfs/h1/ops/prod/com/hafs/v2.0/hfsb.yyyymmdd/hh``
+        - **Disk space usage:** Increased from 145/140 GB (HFSAv1/HFSBv1) per cycle to ~295/195 GB (HFSAv2/HFSBv2).
+        - **HPSS disk requirement:** Increased from ~142/126 GB (HFSAv1/HFSBv1) to ~228/133 GB (HFSAv2/HFSBv2).
+        - **Preferred data retention in COM:** Files to be kept for 7 days.
+
+**Input Datasets**
```

Comment:
This information is already listed above in the input/output section. We should consolidate all input/output info in one place. We usually have an input/output chapter, so it might be good to add info there and link it to it here.